babel.config.js

```diff
@@ -9,4 +9,11 @@ module.exports = {
       { targets: { node: true }, modules: false },
     ],
   ],
+  plugins: [
+    // Force this transform because otherwise Webpack 4 (used in CRA) fails to parse:
```

Comment:
This likely started surfacing due to updating `browserslist`. I guess the positive thing is: That means it's now already trying to not transpile `?.` and `??` but leave it in the bundle :exploding_head: web is moving faster

Reply:
Although TBF I think I saw it happen in a `yarn start` dev build. May be that the production browserslist still has browsers that don't support it.